README.md

```diff
@@ -4,6 +4,6 @@
 
 This is the website landing page for the RSE-eScience-2022 workshop to be
 held in conjunction with [eScience 2022](https://www.escience-conference.org/2022/)
-in Salt Lake City, Utah, October 11-14, 2022.
+in Salt Lake City, Utah, October 10-14, 2022.
```

Comment:
Maybe we should change the date to just the data of the workshop, not the date of the whole conference